descriptors/ModuleDescriptor-template.json

```diff
@@ -667,6 +672,19 @@
         "inn-reach.central-patron-type-mappings.collection.put"
       ]
     },
+    {
+      "permissionName" : "inn-reach.inn-reach-transaction.item.post",
+      "displayName" : "create INN-Reach Transaction with item hold at the owning site",
+      "description" : "Create INN-Reach Transaction with Item Hold at the owning site"
+    },
+    {
+      "permissionName" : "inn-reach.inn-reach-transaction.all",
+      "displayName" : "inn reach API module - all permissions of inn-reach transactions",
+      "description" : "All permissions of inn-reach transactions scope",
+      "subPermissions" : [
+        "inn-reach.inn-reach-transactions.item.post"
```

Comment:
it may be reasonable to have a common prefix for D2IR endpoints. Something like `inn-reach.d2ir.`